### PR TITLE
By default set BIOM table type to "OTU table"

### PR DESCRIPTION
**Pull request by nuada**
Cleaner version of #1937, fix for #1928.

### CODE REVIEW

**Comment by qiime-r2d2** [comment]
Can one of the admins verify this pull request?

**Comment by jairideout** [comment]
ok to test

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1589/

File: qiime/util.py [hunk 1]
```diff
@@ -530,7 +530,7 @@ def get_generated_by_for_biom_tables():
 
 
 def write_biom_table(biom_table, biom_table_fp, compress=True,
-                     write_hdf5=HAVE_H5PY):
+                     write_hdf5=HAVE_H5PY, type='OTU table'):
```
Comment (gregcaporaso) [inline qiime/util.py]: could this be called ``table_type`` to match the cli and because ``type`` is a builtin in python?

Comment (gregcaporaso) [inline qiime/util.py]: could this be called ``table_type`` to match the cli and because ``type`` is a builtin in python?

**Comment by gregcaporaso** [comment]
commits pulled into #1996.

**Comment by gregcaporaso** [comment]
Thanks again for the help with this one @nuada!

**Comment by nuada** [comment]
No problem. Always happy to help.

**Pull request closed without merging**